03app_inriamuseum_showroom/03app_inriamuseum_showroom.c

```diff
@@ -265,7 +267,10 @@ void _ntw_getTime_cb(dn_ipmt_getParameter_time_rpt* reply) {
                 app_vars.step         = STEP_2_US;
                 NRF_RTC0->CC[0]       = RTC0PERIOD_STEP_2_US;
                 trackIdx              = app_vars.moteId-2; // the first mote has moteId 2, yet we want trackIdx 0 for it
-                music_play(SONGTITLE_STAR_WARS,trackIdx);
+                if(f_sw)
+                  music_play(SONGTITLE_STAR_WARS,trackIdx);
```

Comment:
never ever forget to put the `{}`. Many many products have failed becasue of that.